marked-element.html

```diff
@@ -285,6 +285,34 @@
       this.fire('marked-render-complete', {}, {composed: true});
     },
 
+     /**
+     * Fired when the content is being processed and before it is rendered.
+     * Provides an opportunity to implement syntax highlighting, [e.g with highlightjs](https://highlightjs.org/).
```

Comment:
I'm good with the first 2 lines here and the event annotation.

As far as adding the additional detail for using highlightjs with marked-element goes, it may be better to have a dedicated documentation section for this stuff. I will leave that decision up to @e111077.